app/uk/gov/hmrc/hmrcemailrenderer/templates/customs/customsRegistrationSuccessful.scala.txt

```diff
@@ -1,12 +1,23 @@
-@(params: Map[String, Any])@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.SalutationHelper.salutationFrom(params)
+@(params: Map[String, Any])@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.SalutationHelper.saluteFullName(params)
 
-You have successfully registered for customs services.
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.organisationname.OrgNameHelper.orgNameOrYou(params) now registered for Customs Declaration Service (CDS).
 
-Your EORI number is now active and in your customs account. When trading you should only share this number with trusted business partners.
+Active from : @{params.getOrElse("completionDate", "")}
```

Comment:
If there isn't a completionDate, then this will just display an empty string. It might be worth putting something more meaningful, just in case.

Reply:
completionDate should always be populated, but may change this in a few sprints time when we do another journey where it won't be passed